quark/core/quark.py

```diff
@@ -214,6 +216,24 @@ def _evaluate_method(self, method) -> List[List[str]]:
             # for the case of MUTF8String
             instruction = [str(x) for x in instruction]
 
+            # Skip the bytecode that is not analyzed.
+            # e.g. invoke-virtual method+xxxx .
+            if (isinstance(bytecode_obj.parameter, str) and
+                "+" in bytecode_obj.parameter):
+               continue
+
+            # Skip the bytecode that invoke-super without registers.
+            # e.g. 'invoke-super', 'Lorg/apache/commons/net/ntp/TimeInfo;->addComment(Ljava/lang/String;)V'
+            if (instruction[0] == "invoke-super" and
+                len(bytecode_obj.registers) == 0):
+                print(instruction)
+                continue
+
+            # Skip the bytecode that invoke-custom with improper descriptor
+            # e.g. invoke-custom {v14, v0},   Resetting:
+            if instruction[0] == "invoke-custom" and "(" not in bytecode_obj.parameter:
+                continue
```

Comment:
Because those checks are for invalid bytecodes produced by RizinImp, would it be possible do the checks in the `get_method_bytecode` method of RizinImp?

Reply:
Done.